### PR TITLE
Add yearsToDays Function

### PR DESCRIPTION
**Pull request by makstyle119**
Add yearsToDays Function

### CODE REVIEW

**Review by MAKWritingHouse — APPROVED** [review]

**Review by Abdullah-MotiWala — APPROVED** [review]

**Review by muhammad-moiz-takafulbazaar — APPROVED** [review]

**Review by ABKZR — APPROVED** [review]

**Review by bilalalam98 — APPROVED** [review]

**Comment by kossnocorp** [comment]
Can you please rebase the branch?

**Comment by makstyle119** [comment]
> Can you please rebase the branch?

PR is now up to date, thanks for the remainder 😇

File: package.json [hunk 1]
```diff
@@ -5058,6 +5062,10 @@
         "default": "./fp/yearsToMonths.mjs"
       }
     },
+    "./fp/yearsToDays": {
+      "require": "./fp/yearsToDays.js",
+      "import": "./fp/yearsToDays.mjs"
```
Comment (kossnocorp) [inline package.json:5067]:
```suggestion
 "require": {
        "types": "./fp/yearsToDays.d.ts",
        "default": "./fp/yearsToDays.js"
      },
      "import": {
        "types": "./fp/yearsToDays.d.mts",
        "default": "./fp/yearsToDays.mjs"
      }
```

File: package.json [hunk 2]
```diff
@@ -5058,6 +5062,10 @@
...
+    },
```
Comment (kossnocorp) [inline package.json:5068]:
```suggestion
```

**Comment by kossnocorp** [comment]
Shipped with `date-fns@3.1.0`

File: src/yearsToDays/index.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * Convert a number of years to a full number of days.
+ *
+ * @param years - The number of years to be converted
+ *
+ * @returns The number of years converted in days
+ *
+ * @example
+ * // Convert 2 years into days
+ * const result = yearsToDays(2)
+ * //=> 24
```
Comment (bertdeblock) [inline src/yearsToDays/index.ts:18]: Should be `730` I think.

**Pull request merged**